modules/test-runner/src/util/client.ts

```diff
@@ -13,19 +13,18 @@ const wallet = Wallet.fromMnemonic(env.mnemonic).connect(ethProvider);
 
 let clientStore: MemoryStoreService;
 
-export const createClient = async (
-  mnemonic: string = Wallet.createRandom().mnemonic,
-): Promise<IConnextClient> => {
+export const createClient = async (opts?: ClientOptions): Promise<IConnextClient> => {
```

Comment:
shouldn't the type be something like: `opts?: Partial<ClientOptions>` so you can provide a subset of opts to start the client with that would override any defaults?

technically that's happening when setting `clientOpts`, but the typing seems off